subproject-lifecycle.md

```diff
@@ -10,14 +10,16 @@ We recognize 2 options how a subproject can be created in OSG. Either this is a
    - Either get a verbal agreement on a SIG meeting, documented in SIG meeting notes
    - Or file an issue in open-services-group/community with subproject details and get a SIG lead approval on the issue
 2. Define and declare scope of the subproject and end goal
+3. Identify the GitHub repositories owned by the new subproject. Create if needed.
```

Comment:
These repositories are created by Peribolos, see step 4 - update to `github-config.yaml`. Regular org members can't create repos manually, adding the repo entry in github config creates the repos for them.

If there are already repositories for a subproject, it's not a new subproject and user should follow the `Import a workstream to this organization as a subproject` scenario.

Reply:
Right.

The point of adding this here is to highlight the need for supporting repositories. It's about saying that repositories *must* exist for the sub-project.

Without having points 3 and 4 in here, I believe it can be interpreted as that "you do not need a repo to create a sub-project", and I believe this is wrong.

I will try to clarify the language / steps on how to create, i.e. point to the instructions below. But do we agree that having backing repos is a requirement?